packages/termux-tools/build.sh

```diff
@@ -54,30 +54,33 @@ termux_step_make_install() {
 	done
 
 	{
-		echo ""
-		echo "Welcome to Termux!"
-		echo ""
-		echo "Communities: https://termux.dev/community"
-		echo "Gitter chat: https://gitter.im/termux/termux"
-		echo "IRC channel: #termux on libera.chat"
```

Comment:
Any particular reason for removing communities and irc channel?

Reply:
Just for making the motd smaller, but i can include it again if you think it would be better